src/main/java/net/minestom/server/event/player/PlayerChangeHeldSlotEvent.java

```diff
@@ -25,7 +25,7 @@ public PlayerChangeHeldSlotEvent(@NotNull Player player, byte slot) {
     /**
      * Gets the slot which the player will held.
```

Comment:
```suggestion
     * Gets the slot which the player will hold.
```
You didn't change this part, but is this grammar intended 🤔

Reply:
Sorry i dont have the best english but yes thats better